### PR TITLE
pip > pip3🍉

### PR DESCRIPTION
**Pull request by PIYUSH6299**
First contribution to open source.

I did a minimul changes in dockerfile, where i changed pip to pip3.

### CODE REVIEW

**Comment by k-boikov** [comment]
Reasoning? I currently do not see any problems.

**Comment by monoy00** [comment]
Sorry, i missclick something. I am new and don’t know what I do…


Von: k-boikov ***@***.***>
Gesendet: Freitag, 21. April 2023 00:55
An: Significant-Gravitas/Auto-GPT ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: Re: [Significant-Gravitas/Auto-GPT] pip > pip3🍉 (PR #2716)


Reasoning? I currently do not see any problems.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Development of this free, open-source project is made possible by all the <a hre
...
...sp;&nbsp;<a href="https://github.com/Nalhos"><img src="https://github.com/Nalhos.png" width="50px" alt="Nalhos" /></a>&nbsp;&nbsp;<a href="https://github.com/Kazamario"><img src="https://github.com/Kazamario.png" width="50px" alt="Kazamario" /></a>&nbsp;&nbsp;<a href="https://github.com/pingbotan"><img src="https://github.com/pingbotan.png" width="50px" alt="pingbotan" /></a>&nbsp;&nbsp;<a href="https://github.com/indoor47"><img src="https://github.com/indoor47.png" width="50px" alt="indoor47" /></a>&nbsp;&nbsp;<a href="https://github.com/AuroraHolding"><img src="https://github.com/AuroraHolding.png" width="50px" alt="AuroraHolding" /></a>&nbsp;&nbsp;<a href="https://github.com/kreativai"><img src="https://github.com/kreativai.png" width="50px" alt="kreativai" /></a>&nbsp;&nbsp;<a href="https://github.com/hunteraraujo"><img src="https://github.com/hunteraraujo.png" width="50px" alt="hunteraraujo" /></a>&nbsp;&nbsp;<a href="https://github.com/Explorergt92"><img src="https://github.com/Explorergt92.png" width="50px" alt="Explorergt92" /></a>&nbsp;&nbsp;<a href="https://github.com/judegomila"><img src="https://github.com/judegomila.png" width="50px" alt="judegomila" /></a>&nbsp;&nbsp;
```
Comment (Pwuts) [inline README.md:49]: Nope.

File: Dockerfile [hunk 2]
```diff
@@ -29,7 +29,7 @@ USER appuser
 # Copy the requirements.txt file and install the requirements
 COPY --chown=appuser:appuser requirements.txt .
 RUN sed -i '/Items below this point will not be included in the Docker Image/,$d' requirements.txt && \
-	pip install --no-cache-dir --user -r requirements.txt
+	pip3 install --no-cache-dir --user -r requirements.txt
```
Comment (Pwuts) [inline Dockerfile:32]: unnecessary, `pip` works fine

**Comment by PIYUSH6299** [comment]
I'm just new here 
Don't know the rules and regulation 
Sorry folks to interrupt you all.

**Comment by PIYUSH6299** [comment]
> Reasoning? I currently do not see any problems.

Yes you are right but i thought pip3 will work better.

**Comment by Pwuts** [comment]
> I'm just new here

Just a bit of advice then: don't try to sneak yourself into the sponsor gallery while claiming to make only a minimal change to the Dockerfile. We appreciate the interest, but do not have the capacity to deal with this kind of nonsense.

**Comment by PIYUSH9090** [comment]
> > I'm just new here
> 
> Just a bit of advice then: don't try to sneak yourself into the sponsor gallery while claiming to make only a minimal change to the Dockerfile. We appreciate the interest, but do not have the capacity to deal with this kind of nonsense.

Okay :(

**Pull request closed without merging**